test/services/apimap-fields-formater.test.js

```diff
@@ -18,8 +13,9 @@ describe('Service > Apimap Fields Formater', () => {
   }], 'Users').perform();
 
   it('should filter fields without declared "field" attribute', () => {
-    expect(fieldsFormated.length).equal(2);
-    expect(fieldsFormated).to.containSubset([{
+    expect.assertions(2);
```

Comment:
[Misc] I'm not sure to understand what's the purpose of `expect.assertions` when there is no asynchronous code? Still, maybe you use it to be consistent ("all test code MUST start with `expect.assertions`"). I'm OK with this, I'm just curious!

Reply:
No really clear in the rule documentation:
https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/prefer-expect-assertions.md

But consistency is probably the main reason.